composer.json

```diff
@@ -49,9 +49,15 @@
             "web/themes/contrib/{$name}": ["type:drupal-theme"],
             "drush/contrib/{$name}": ["type:drupal-drush"]
         },
-        "enable-patching": true
+        "enable-patching": true,
+        "patches":{
+            "asm89/stack-cors": {
+                "(November 29 2017 06:22:06 PM: See issue #223 on contentacms/contenta_jsonapi) Add origin matcher for wildcard matching": "patches/asm89_stack-cors/pull_request_42.patch"
+            }
+        }
     },
     "config": {
+        "preferred-install": "source",
```

Comment:
Why this change?

Reply:
If this is related to the git pulling of the cors library, can we do:
```
        "preferred-install": {
            "asm89/stack-cors": "source",
            "*": "auto"
        }
```

Reply:
Yeah, preferred install source slows down the install process quite a bit.

Reply:
fixed via https://github.com/contentacms/contenta_jsonapi_project/commit/c7d88b10e5dc78427ac29ff1a036cf2b09ecbf70